server/src-lib/Hasura/RQL/DDL/Schema.hs

```diff
@@ -98,11 +100,11 @@ runRunSQL RunSQL {..} = do
           let e = err400 PostgresError "query execution failed"
            in e {qeInternal = Just $ toJSON txe}
 
-    isAltrDropReplace :: QErrM m => T.Text -> m Bool
-    isAltrDropReplace = either throwErr return . matchRegex regex False
+    containsMutationKeyword :: QErrM m => T.Text -> m Bool
```

Comment:
I think something along the lines of `containsDDLKeyword` might be better?

Reply:
I agree, and I’ve made this change.